polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -150,4 +152,17 @@ public static <T> Builder<T> builder() {
               "If set to true, allows tables to have external locations outside the default structure.")
           .defaultValue(false)
           .build();
+
+  public static final PolarisConfiguration<List<String>> SUPPORTED_CATALOG_STORAGE_TYPES =
```

Comment:
Why not `PolarisConfiguration<List<StorageConfigInfo.StorageTypeEnum>>`?

Reply:
These configurations are all basic types that DropWizard knows how to parse.

Reply:
So Dropwizard cannot parse enums? 🤦‍♂️

Reply:
It seems to have trouble parsing even to things like Boolean, so I'm not sure. If we do change this later, it shouldn't be a user-facing change in any event